<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #202020;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="open_file">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;
	background-color: grey;
	border-style: outset;
    border-width: 0.5px;
	border-color: white;
}

QPushButton:pressed {
	color: black;
	background-color: white;
	border-style: outset;
     border-width: 0.5px;
	border-color: black;
}</string>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>30</y>
      <width>940</width>
      <height>800</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
  background: #383838;
}

QTabBar::tab {
  font-size: 18px;
  background: gray;
  color: white;
  padding: 5px;
 }

 QTabBar::tab:selected {
  color: black;
  background: lightgray;
 }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="maze">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Maze</string>
     </attribute>
     <widget class="QPushButton" name="generate_maze">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>670</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	background-color: grey;
	border-style: outset;
    border-width: 0.5px;
	border-color: white;
}

QPushButton:pressed {
	color: black;
	background-color: white;
	border-style: outset;
     border-width: 0.5px;
	border-color: black;
}</string>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_height">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>635</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Height of Maze</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="height_maze">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>670</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	color: white;
	background-color: grey;
}</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="displayIntegerBase">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="name_width">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>635</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Width of Maze</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="width_maze">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>670</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	color: white;
	background-color: grey;
}</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="cave">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Cave</string>
     </attribute>
     <widget class="QPushButton" name="auto_mode">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>675</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	background-color: grey;
	border-style: outset;
    border-width: 0.5px;
	border-color: white;
}</string>
      </property>
      <property name="text">
       <string>Auto mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="name_lim2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>640</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Limit of Birth</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="limit_death">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>670</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	color: white;
	background-color: grey;
}</string>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLabel" name="name_lim1">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>640</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Limit of Death</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="for_auto_timer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>625</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QDoubleSpinBox {
	color: white;
	background-color: grey;
}</string>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>50000.000000000000000</double>
      </property>
      <property name="value">
       <double>3000.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="next_step">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>675</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: white;
	background-color: grey;
	border-style: outset;
    border-width: 0.5px;
	border-color: white;
}

QPushButton:pressed {
	color: black;
	background-color: white;
	border-style: outset;
     border-width: 0.5px;
	border-color: black;
}</string>
      </property>
      <property name="text">
       <string>Next step</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="limit_birth">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>670</y>
        <width>100</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
	color: white;
	background-color: grey;
}</string>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="Widget" name="area" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>670</y>
      <width>880</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
	background-color: grey;
}</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>open_file</zorder>
   <zorder>area</zorder>
   <zorder>widget</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QWidget</extends>
   <header location="global">widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
